docker/build.mk

```diff
@@ -22,11 +22,13 @@ BASE_TAG ?= gcr.io/fuzzbench
 build-all: $(addsuffix -all, $(addprefix build-,$(FUZZERS)))
 pull-all: $(addsuffix -all, $(addprefix pull-,$(FUZZERS)))
 
+# If we're running using GitHub Workflows, cache-from a remote image. Otherwise
+# just use the local cache.
+cache_from = $(if ${GITHUB_RUN_ID},--cache-from $(1),)
```

Comment:
please add travis as well

Reply:
I've changed it to where we set the env var in `build.py` so as long as we use that for Travis or other CI's, it should be fine.

I'm watching the test run below to ensure the env var is properly propagated though.